test/fail_compilation/pragmainline2.d

```diff
@@ -16,15 +16,15 @@ void foo()
     pragma(inline, false);
     pragma(inline);
     pragma(inline, true);   // this last one will affect to the 'foo'
-    while (0) { }
+    asm { nop; }
 }
 
-pragma(inline, true)   void f1t() { while (0) {} }  // cannot inline
-pragma(inline, false)  void f1f() { while (0) {} }
-pragma(inline)         void f1d() { while (0) {} }
-void f2t() { pragma(inline, true);  while (0) {} }  // cannot inline
-void f2f() { pragma(inline, false); while (0) {} }
```

Comment:
Sorry I'm not sure what you mean.

The point of these test case changes is, the `while (0) {}` statements can be inlined after this PR at the caller site of `f1t` and `f2t` functions.
By the inliner improvement, the original code couldn't test `pragma(inline, true)` behavior anymore. Therefore I replaced the `while` loops with `asm` statements, because `asm` never be inline-able.